org.testeditor.web.dropwizard.xtext/src/main/java/org/testeditor/web/xtext/index/BuildCycleManager.xtend

```diff
@@ -118,6 +160,10 @@ class ChunkedResourceDescriptionsProvider implements IResourceDescriptionsProvid
 		return project.get
 	}
 
+	def ResourceDescriptionsData getData() {
```

Comment:
getData seems a bit short/generic for this method. Why not name it get(Project)ResourceDescriptionsData